rls/src/main/java/io/grpc/rls/SubchannelStateManagerImpl.java

```diff
@@ -30,6 +30,7 @@
 
   private final HashMap<String, ConnectivityState> stateMap = new HashMap<>();
   private final Multiset<ConnectivityState> stateMultiset = HashMultiset.create();
+  private ConnectivityState currentState;
```

Comment:
Just in case - does this need to be volatile? `getAggregatedState()` now reads and writes to the `currentState`, so it's not idempotent anymore. Or that was the point?